mqmetric/queue.go

```diff
@@ -0,0 +1,351 @@
+/*
+Package mqmetric contains a set of routines common to several
+commands used to export MQ metrics to different backend
+storage mechanisms including Prometheus and InfluxDB.
+*/
+package mqmetric
+
+/*
+  Copyright (c) IBM Corporation 2016, 2018
```

Comment:
If this is a new file the copyright start date should be 2018 not 2016

Reply:
Done